ec-europa/eubfr-data-lake
# feat(etl): add FTS - EUBFR-202

# PR description

Steps to test:
- update `config.json` with credentials for the new producer
- `yarn` and `yarn deploy`
- use `npx eubfr-cli content upload .content/fts/export_2017.xlsx -p fts` or manual upload
- verify mapping to actual data

Data is available here: https://data.europa.eu/euodp/data/dataset/FTS

Failing tests are not related to updates in this pull request.

## QA Checklist

When you add a new ETL/producer, please check for the following:

* [ ] Producer's secrets are stored safely
* [ ] Ensure the ETL is added to the corresponding `scripts/` for automated deployment and deletion 
* [ ] There is at least 1 unit test with a jest snapshot for the transform function of the ETL
* [ ] Update the file PRODUCERS_DATA_AVAILABILITY_GRID.md indicating which fields are available in source data of ETL
* [ ] Ensure there is (flow/jsdocs) documentation in the `tranform.js` file which is to be used for automated documentation
* [ ] Generate the necessary documentation pages for the new ETL by `yarn docs:md`
